src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs

```diff
@@ -1615,8 +1615,7 @@ private void MakeSeparatorList(ReadOnlySpan<char> separators, ref ValueListBuild
                 sep0 = separators[0];
                 sep1 = separators.Length > 1 ? separators[1] : sep0;
                 sep2 = separators.Length > 2 ? separators[2] : sep1;
-
-                if (Length >= 16 && Sse41.IsSupported)
+                if (Vector128.IsHardwareAccelerated && Length >= Vector128<ushort>.Count * 2)
```

Comment:
Just to double-check: the `* 2` is intentional as perf-numbers showed that?

Reply:
Yes exactly, smaller strings doesn't perform as well